### DISCUSSION TITLE
Getting errors after update. Nothing shows up on Kibana.

### DISCUSSION THREAD

**Discussion by Infra-L** [Security-Onion-Solutions/securityonion#9737] [Unsupported Versions]
Would appreciate it if somebody could point me in the right direction. When I try to open any dashboards on Kibana I get error messages. I included examples below.
![2](https://user-images.githubusercontent.com/124704173/217294425-99288616-f66d-4111-ab36-d2c1b5f2f435.PNG)
![1](https://user-images.githubusercontent.com/124704173/217294429-c52b196a-15aa-4602-aa19-bea0333297cc.PNG)
![3](https://user-images.githubusercontent.com/124704173/217294431-68cd054d-ba98-41b9-a649-a58cf69dbfbd.PNG)

**Comment by cm-ops** [comment]
If you would provide some information around your grid - https://github.com/Security-Onion-Solutions/securityonion/discussions/1720

The first screen grab leads me to think you are opening a `z16.04` dashboard, those are legacy from 16.04. Your indices should be `*:so-*` If you try the `Security Onion - Home` dashboard does it load?
